### PR TITLE
Changed command display style to table in .md files

### PR DESCRIPTION
**Pull request by VaiTon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Gringotts
 
 Gringotts is an item-based economy plugin for the Bukkit Minecraft server platform. Unlike earlier economy plugins, all currency value and money transactions are based on actual items in Minecraft, per default emeralds. The goals are to add a greater level of immersion, a generally more Minecraft-like feeling, and in the case of a PvP environment, making the currency itself vulnerable to raiding.
 
-[Get Gringotts from BukkitDev](http://dev.bukkit.org/server-mods/gringotts/)
+[Get Gringotts from Spigot](https://www.spigotmc.org/resources/gringotts.42071/)
```
Comment (jastice) [inline README.md]: BukkitDev is still the "official" distribution. But we could link to both sites from the readme.

File: doc/usage.md [hunk 2]
```diff
@@ -39,31 +39,22 @@ The permission `createvault.forothers` allows you to create vaults for other pla
...
-
-    /money withdraw <amount>
-Withdraw an amount from chest storage into inventory.
+Arguments between `<>` (ex `<amount>`) are mandatory while ones between `[]` (ex. `[type]`) are optionals.
 
-    /money deposit <amount>
-Deposit an amount from inventory into chest storage. 
+### User commands ###
 
+| Command           | Arguments           | Description                             | Aliases         |
```
Comment (jastice) [inline doc/usage.md]: I think the separate columns for command and arguments make the command structure less clear

File: README.md (hunk 1 above)
Comment (VaiTon) [reply README.md]: That link is now a 404 so it should be changed anyway

File: doc/usage.md (hunk 2 above)
Comment (VaiTon) [reply doc/usage.md]: Hmm, ok you're right, i'll combine them.

File: README.md (hunk 1 above)
Comment (jastice) [reply README.md]: Yes, the new link is https://dev.bukkit.org/projects/gringotts

Comment (VaiTon) [reply README.md]: Done 👍

**Comment by jastice** [comment]
Thanks!

**Pull request merged**